observability/otel/otel-collector/otel-collector-connectors.adoc

```diff
@@ -78,3 +78,8 @@ include::snippets/technology-preview.adoc[]
 # ...
 ----
 <1> Defines the flush interval of the generated metrics. Defaults to `15s`.
+
+[role="_additional-resources"]
+[id="additional-resources_otel-collector-connectors_{context}"]
+== Additional resources
+* link:https://opentelemetry.io/docs/specs/otlp/[OpenTelemetry Protocol (OTLP) specification]
```

Comment:
```suggestion
* link:https://opentelemetry.io/docs/specs/otlp/[OpenTelemetry Protocol (OTLP) specification (OpenTelemetry documentation)]
```

Consider this extra information to give a heads-up to customers that they will be going to a docs set outside the customer portal.

Reply:
Thank you for the suggestion. To keep it simple, I replaced `specification` with `documentation`.